src/EPR.RegulatorService.Facade.Core/Services/Application/ApplicationService.cs

```diff
@@ -24,11 +25,16 @@ public class ApplicationService : IApplicationService
 
     public async Task<HttpResponseMessage> PendingApplications(Guid userId, int currentPage, int pageSize, string organisationName, string applicationType)
     {
-        var url = string.Format($"{_config.Endpoints.PendingApplications}", userId, currentPage, pageSize, organisationName, applicationType);
-
         _logger.LogInformation("Attempting to fetch pending applications from the backend");
 
-        return await _httpClient.GetAsync(url);
+        var url = string.Format($"{_config.Endpoints.PendingApplications}", userId, currentPage, pageSize, organisationName, applicationType);
```

Comment:
Please undo this change, the uriBuilder.Path returns encoded Url and if there is ? in the Url it will be converted to 3f something, ignore the SQ warning as peter is happy to leave it as is